themes/bootstrap3/templates/RecordDriver/DefaultRecord/list-entry.phtml

```diff
@@ -188,7 +188,7 @@
             <?=$this->transEsc('Delete') ?>
           </a>
           <ul class="dropdown-menu" role="menu" aria-labelledby="<?=$dLabel ?>">
-            <li><a onClick="$.post('<?=$deleteUrl?>', {'delete':'<?=$this->escapeJs($id) ?>','source':'<?=$this->escapeJs($source) ?>','confirm':true},function(){location.reload(true)})" title="<?=$this->transEsc('confirm_delete_brief')?>"><?=$this->transEsc('confirm_dialog_yes')?></a></li>
+            <li><a id="confirm_delete_item_<?=$this->escapeHtml($id);?>" title="<?=$this->transEsc('confirm_delete_brief')?>"><?=$this->transEsc('confirm_dialog_yes')?></a></li>
```

Comment:
Even with proper escaping, I'm concerned that there might be weird ID values that could create HTML-illegal IDs. How would you feel about using an md5 of the ID as the element ID, just to be on the safe side?

Reply:
I do think, it is not proper escaping, it should be `escapeHtmlAttr`, I tried to change, does it work for you now?

Reply:
Great, thanks! I was mainly concerned about very strange IDs breaking the logic, but based on [this](https://mathiasbynens.be/notes/html5-id-class) it sounds like that's probably not a major concern. :-)